integration/network/ipvlan/ipvlan_test.go

```diff
@@ -425,7 +429,26 @@ func testIpvlanAddressing(client dclient.APIClient) func(*testing.T) {
 	}
 }
 
-// ensure Kernel version is >= v4.2 for ipvlan support
+var (
+	once            sync.Once
+	ipvlanSupported bool
+)
+
+// figure out if ipvlan is supported by the kernel
 func ipvlanKernelSupport() bool {
-	return n.CheckKernelMajorVersionGreaterOrEqualThen(4, 2)
+	if n.CheckKernelMajorVersionGreaterOrEqualThen(4, 2) == false {
```

Comment:
Generally we don't compare with a boolean, so probably;

```suggestion
	if !n.CheckKernelMajorVersionGreaterOrEqualThen(4, 2) {
```

Perhaps we should add a `CheckKernelMajorVersionLowerThan`

Reply:
Ah, looks like the linter found it as well 

```
21:11:48 integration/network/ipvlan/ipvlan_test.go:439:5:warning: should omit comparison to bool constant, can be simplified to !n.CheckKernelMajorVersionGreaterOrEqualThen(4, 2) (S1002) (gosimple)